### PR TITLE
Added polkasearch application

### PR DESCRIPTION
**Pull request by avirajkhare00**
#### Project Abstract

Polkasearch.xyz aims to become a discovery platform where users can discover Grants, Bounties, Jobs, Tools, Educational material for Polkadot/Substrate, all under one roof.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) address for the payment of the milestones is provided in the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by avirajkhare00** [comment]
Hey @Noc2. `ink-smart-contract-wizard` is completed. You can check here: https://github.com/w3f/Grant-Milestone-Delivery/pull/691

File: applications/polkasearch.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# polkasearch.xyz
+
+* **Proposer:** Aviraj Khare
```
Comment (keeganquigley) [inline applications/polkasearch.md:3]:
```suggestion
* **Team Name:** Aviraj Khare
```

**Review by keeganquigley — CHANGES_REQUESTED** [review]
@avirajkhare00 thanks for the new application. A few initial questions for you.

- Can you please add more detail to the deliverables tables? For example, what languages do you plan to use? What tools/frameworks will you use for frontend/backend? How will the CRM be built? Will you use databases, smart contracts, etc.
- Under the [Overview section](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#overview-1), can you please add the **Total Estimated Duration, Full-time Equivalent (FTE), and Total Costs?**
- Regarding M1, in my opinion 4k seems like a lot for a simple login/signup system. I would recommend combining this with M2 and reducing the price (if you reduce it to 10k it will only need 2 approvals). 
- Also, are you going to include the ability to connect to an extension like Polkadot{JS}, Talisman, etc? I personally wouldn't approve a traditional Web2 signup OAuth scheme, unless it's combined with Web3 connect options.
- Regarding the very last M4 deliverable, we typically only fund tech/development and not educational materials. Can you remove this or define what modules you would be building for this part?

**Comment by avirajkhare00** [comment]
Hey @keeganquigley. I have addressed all the issues in the grant application. Requesting to review it again 🙏

**Review by keeganquigley — DISMISSED** [review]
thanks @avirajkhare00 it looks a lot better now. Considering the scope of work I'm happy to accept it, and I will share it internally with the rest of the committee.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. Do you already have some mock-ups or designs that you can share? Usually, for frontend-focused applications, we ask teams to provide some initial designs upfront (see https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#project-details). Also, are you aware of https://dotjobs.net/?

**Comment by avirajkhare00** [comment]
Hey @Noc2. I have added a few designs. Please check 🙏 
No sir, I wasn't aware of https://dotjobs.net

**Review by keeganquigley — DISMISSED** [review]
@avirajkhare00 thanks for the updates, I agree it would be especially nice to have a place where we can keep track of all the grants programs for the entire ecosystem.

**Comment by avirajkhare00** [comment]
Hey @Noc2. Can you please review it 🙏

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for sharing the initial designs. But to be honest, I have a lot of doubts when it comes to this application. The biggest is that it will be very hard to convince others actually to use this system. How do you plan to integrate existing solutions or convince users actually to use your solution? Also, does your website get the bounty information from polkadot or kusama or do you mean a separate bounty program? One reason why I want to double-check this is also because you need some initial data on this platform so that people will start using it.

**Comment by avirajkhare00** [comment]
> Thanks for sharing the initial designs. But to be honest, I have a lot of doubts when it comes to this application. The biggest is that it will be very hard to convince others actually to use this system. How do you plan to integrate existing solutions or convince users actually to use your solution? Also, does your website get the bounty information from polkadot or kusama or do you mean a separate bounty program? One reason why I want to double-check this is also because you need some initial data on this platform so that people will start using it.

@Noc2 you are correct here. To begin with, If I leave the website running, no one will come and check it out.
To begin with, my team needs to fill out Grants, Jobs, Tools, and Educational Materials ourselves. We will be reaching out to companies in polkadot ecosystem and ask them if they have job opening or bounties available. We will fill it out ourselves. I am personally part of a lot of Telegram and WhatsApp Blockchain groups. Initially, we need to push this out to the users.

To answer your question, to begin with, we need to populate the data manually. We will be looking at bounties in Polkadot ecosystem and will curate them here. We don't have any intention to run our own bounty program till we have enough traction.

This is how superteam.fun started. They started to curate the data around Solana ecosystem manually.
Now they have 30k twitter followers, 40k+ discord members.

I have a call with Nicholas from Parity who is looking SquareOne initiative. He is very much interested about this since I will be working part-time on it after building this. Once we have some traction, then I can push it to Substrate builders program.

**Comment by avirajkhare00** [comment]
Alchemy has dApp store. For all the blockchains: https://www.alchemy.com/dapps

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the quick reply. Could you add to the deliveries that you initially will add x grants, bounties, and jobs manually?

**Comment by avirajkhare00** [comment]
@Noc2 @keeganquigley I have updated the application. I have given the minimum target to reach for every category. However, I haven't seen the Bounties in this ecosystem so can't commit to the number. However, we will try our best to reach companies/protocol building on the Polkadot ecosystem.

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I'm happy to go ahead with it.

**Comment by gmajor-encrypt** [comment]
@Noc2 Hi, I think this project is not worthy of donations, it is a simple web application, packaged into ``Polkadot/substrate`` related. Currently, https://forum.polkadot.network/ is already strong enough. 

Although it also includes some of the features described in this proposal, it is very simple to implement these features using Django, so I think the price is too high

The community needs a place where all information can be aggregated, not scattered everywhere.

**Comment by avirajkhare00** [comment]
Hey @gmajor-encrypt. Thanks for the review. The word "simple" is relative here. It depends on person to person. You might be very professional in Django, so that it might be very simple for you.

Can you point me in this doc that says "a simple web application" can't be part of a grant: https://github.com/w3f/Grants-Program.

> The community needs a place where all information can be aggregated, not scattered everywhere.

Can you elaborate more on this?

Thanks

**Comment by gmajor-encrypt** [comment]
@avirajkhare00 Hey, 
1. First I just express my opinion, which does not affect whether the project can be accepted
2. I agree with your work calculation, sorry to bring up my own situation
3. The project is more like a traditional centralized community project, which requires someone to operate it, so there is no meaning of open source. As I mentioned above, everyone uses the open-source code of this project to build many similar community sites. Meaning, we need a centralized place to find enough information

**Review by randombishop — APPROVED** [review]
@avirajkhare00 thank you for this application, I think the scope and budget are reasonable. 
And if you fill up the website with quality data, and work hard on marketing over social media, the website could be a successful platform for the community.
So I am happy to go ahead with it.

**Comment by semuelle** [comment]
Hi @avirajkhare00, would you mind sharing an update on the project? According to my calculations, you were planning to submit M1 in early March. Feel free to [amend the timeline](https://w3f.github.io/Grants-Program/docs/Process/changes) in the grant if you foresee longer delays.

**Comment by avirajkhare00** [comment]
Hey @semuelle. Sure. There are few changes in the timeline which i will update soon.

**Comment by keeganquigley** [comment]
Hi @avirajkhare00 friendly reminder. Do you still plan to submit an amendment soon?

**Comment by keeganquigley** [comment]
pinging @avirajkhare00

**Comment by keeganquigley** [comment]
Hi @avirajkhare00 I am not finding a repo for the project, and I also noticed that you have very little activity on your GH account as of late. Considering this, I think it's probably best to close the grant for now, and it can potentially be re-opened in the future should you want to continue working on it. 

Please note that if we don't hear from you after 2 weeks I will submit a cancellation PR for the committee to review. Let me know if you have any questions.

**Comment by avirajkhare00** [comment]
Hey @keeganquigley. I am ready to work on this grant now. I have two questions:
1. Do I need to increase the estimated duration of Milestones 1 and 2 since it is already delayed due to personal reasons?
2. I need to change the payment address because I want USDC instead of BTC.

**Comment by Noc2** [comment]
@avirajkhare00 Thanks for pinging us here. Feel free to create a quick PR to update the contract. That is usually the easiest and safest.

**Comment by keeganquigley** [comment]
Thanks @avirajkhare00:

1. Yes that would be great; feel free to create an amendment PR to extend the timeline.
2. You can change this as well in the same PR. Just make sure to have the same USDC address as the one you provide on your invoice. Thanks!

**Comment by avirajkhare00** [comment]
Hi. @Noc2 @keeganquigley since I am not working on polkasearch due to bandwidth issue. I am happy to sell domain if anyone is interested.
Domain name is: polkasearch.xyz
If anyone in community wants to buy

**Pull request merged**